PennMobile/Setup + Navigation/RootViewController.swift

```diff
@@ -133,6 +128,25 @@ class RootViewController: UIViewController, NotificationRequestable {
             
         }
         
+        //Request to fetch Two Factor Code again if the app failed to fetch it last time
+        if shouldFetchTwoFactorCode() {
+            DispatchQueue.main.async {
+                let alert = UIAlertController(title: "Two-Step Code Not Fetched", message: "We were unable to fetch your Two-Step code last time. Do you want to try again?", preferredStyle: .alert)
+                alert.addAction(UIAlertAction(title: "No", style: .default, handler: { _ in
+                    alert.dismiss(animated: true, completion: nil)
+                    UserDefaults.standard.setTwoFactorEnabled(to: false);
+                }))
+                
+                alert.addAction(UIAlertAction(title: "Yes", style: .default, handler: { _ in
+                    let twc = TwoFactorWebviewController()
+                    UserDefaults.standard.setTwoFactorEnabledAgain(to: true)
+                    self.current.present(twc, animated: true)
```

Comment:
Why are we showing a webview controller here instead of just making a request directly? Do we need them to login again?

Reply:
This is something you had mentioned - that we don't want to fetch the user's code without making them login when they reopen the app - to make it clear that we have the user's permission.